utilities/getPageMetadata.ts

```diff
@@ -49,13 +54,15 @@ export const getPageMetadata = ({
 }: PageMetadataProps) => {
   // We only care about canonical URL for production so ok to use the prod URL here
   const { siteTitle, siteDescription, siteUrlProd: siteUrl } = config;
+  const heroPickerImage = heroPicker?.[0]?.content?.heroImage?.filename;
+  const heroPickerFocus = heroPicker?.[0]?.content?.heroImage?.focus;
 
   const title = seoTitle || pageTitle;
   const searchTitle = slug === 'home' ? 'Home' : title;
   const description = seoDescription || dek || siteDescription;
   const ogTitle = og_title || title;
   const ogDescription = og_description || description;
-  const heroImageCropped = getProcessedImage(filename || bgFilename, '1200x630', focus || bgFocus);
+  const heroImageCropped = getProcessedImage(filename, '1200x630', focus) || getProcessedImage(bgFilename, '1200x630', bgFocus) || getProcessedImage(heroPickerImage, '1200x630', heroPickerFocus);
```

Comment:
Story filter pages don't have individual  hero or background hero images, so using the image from the hero picker as default for OG image